wnfs/src/private/keys/exchange.rs

```diff
@@ -138,6 +203,80 @@ mod test {
     }
 
     #[async_std::test]
+    async fn test_rsa_priv_key_from_pem_file() {
```

Comment:
nit: you can have your test functions return Results such that you dont need to use as many unwraps. just less cluttered when using ?'s instead

Reply:
consider it done